app/ios/.gitignore

```diff
@@ -0,0 +1,32 @@
+*.mode1v3
```

Comment:
Like my comment before, can these all be merged into one .gitignore? Maybe not, but worth playing with to see what breaks, if anything.